### PR TITLE
Expand metric units to support more types of products

### LINKED ISSUE #13002: Add more units to numeric fields
Expand metric units to support more types of products.

Add metric units:

Metric -> Distance -> mm
Metric -> Distance -> dm
Metric -> Area -> mm2
Metric -> Area -> dm2

### PR DESCRIPTION
**Pull request by FremahA**
I want to merge this change because it expands metric units to support more types of products.

Fixes #13002

# Impact

* [* ] New migrations
* [ ] New/Updated API fields or mutations
* [ ] Deprecated API fields or mutations
* [ ] Removed API types, fields, or mutations
* [ ] Documentation needs to be updated

# Pull Request Checklist

* [ ] Privileged queries and mutations are either absent or guarded by proper permission checks
* [ ] Database queries are optimized and the number of queries is constant
* [ ] Database migrations are either absent or optimized for zero downtime
* [ ] The changes are covered by test cases

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -76,6 +76,7 @@ All notable, unreleased changes to this project will be documented in this file.
 - Match `Orders` to `User` when creating user using OIDC plugin. - #12863 by @kadewu
 - Allow defining a custom price in draft orders - #12855 by @KirillPlaksin
 - Update price resolvers - use `discounted_price` on `ProductChannelListing` and `ProductVariantChannelListing` channel listings to return the pricing - #12713 by @IKarbowiak
+- Expand metric units to support more types of products. - #13043 by @FremahA
```
Comment (maarcingebala) [inline CHANGELOG.md:79]: Please move this entry to 3.15 section.

**Review by maarcingebala — APPROVED** [review]

**Review by IKarbowiak — APPROVED** [review]

**Comment by maarcingebala** [comment]
Thanks for contributing @FremahA!

**Comment by maarcingebala** [comment]
Great to hear! There is a bunch of issues that are relatively simple or with a rather small scope that you can look for; they're marked as [good first issue](https://github.com/saleor/saleor/labels/good%20first%20issue). Others that we would appreciate helping with are marked as [bug](https://github.com/saleor/saleor/labels/bug).

**Comment by FremahA** [comment]
Cool. I will take a look at the issues.

**Pull request merged**